NangPaGo-be/src/main/java/com/mars/NangPaGo/domain/ingredient/controller/IngredientElasticController.java

```diff
@@ -0,0 +1,30 @@
+package com.mars.NangPaGo.domain.ingredient.controller;
+
+import com.mars.NangPaGo.domain.ingredient.service.IngredientElasticSynchronizer;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RequiredArgsConstructor
+@RequestMapping("api/ingredient")
```

Comment:
질문: URI 접두사로 `api` 가 필요한 이유가 있을까요? 굳이 필요없다면, 다른 코드와 맞추기 위해 지우는게 나을 것 같아요.

컨벤션: URI를 지정할때는 맨 앞에 `/`를 붙여주세요. (실제 동작에 차이는 없다고 하네요.)
```suggestion
@RequestMapping("/api/ingredient")
```

Reply:
ae8873357ed41002ff351865169e9642726a8bf6 👌🏻 반영 완료했습니다~